1.8/usage/jobs/getting-started.md

```diff
@@ -59,11 +165,19 @@ curl -H "Authorization: token=$(dcos config show core.dcos_acs_token)" $(dcos co
 The following command lists job runs:
 
 ```
-curl -H "Authorization: token=$(dcos config show core.dcos_acs_token)" “$(dcos config show core.dcos_url)/service/metronome/v1/jobs/myjob/runs/“
```

Comment:
WhenI run the command below as follows I get an empty result:
```
calico:~ suzanne$ curl -H "Authorization: token=$(dcos config show core.dcos_acs_token)" $(dcos config show core.dcos_url)/service/metronome/v1/jobs/myjob/runs/
{"message":"","requestPath":"/v1/jobs/myjob/runs/"}
```
Does this mean that the job isn't currently running? I'm assuming if I run this and a job _is_ running, I'd enter the run ID where "20160725212507ghwfZ" is in the `stop run` command below?